### PR TITLE
feat: [console-lite] - trading - order review

### LINKED ISSUE #352: Ability Review and Submit my order
## Task
As a user I would like to Review and Submit my order

## ACs

- [x] It should show my direction of the trade
- [x] It should show the name of the market
- [x] It should show the expiration date
- [x] It should show the execution price
- [x] It should show the estimated margin
- [x] It should show the size
- [x] It should show the estimated close out price
- [x] It should have a button to submit the order
- [x] It should give me an indication that I need to sign the transaction in my wallet
- [x] It should match the design
- [x] It should work on dark/light mode
- [x] It should read out on screen readers and be A11Y friendly

### PR DESCRIPTION
**Pull request by macqbat**
# Related issues 🔗

Closes #352

# Description ℹ️

Order summary view before sending it

# Demo 📺

image

image

# Technical 👨‍🔧

a couple of hooks for calculating estimated margins and close out values

### CODE REVIEW

File: apps/simple-trading-app/src/app/hooks/use-order-closeout.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    const price = parseFloat(
+      addDecimal(market.depth.lastTrade.price, market.decimalPlaces)
+    );
+    const balance = parseFloat(
+      addDecimal(account.balance, account.asset.decimals)
+    );
+    const { size, side } = order;
+    return formatNumber(
+      side === VegaWalletOrderSide.Buy
+        ? (1 - balance / (+size * price)) * price
```
Comment (mattrussell36) [inline apps/simple-trading-app/src/app/hooks/use-order-closeout.ts:29]: suggestion (non blocking): I'd probably do the math in this hook with BigNumber

File: apps/simple-trading-app/src/app/hooks/use-market-positions.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  return markets.length
+    ? markets.reduce(
+        (agg, item) => {
+          const balance = item.market.accounts?.reduce(
+            (acagg, account) => acagg + (+account.balance || 0),
+            0
+          );
+          if (balance) {
+            agg.balanceSum += balance;
```
Comment (mattrussell36) [inline apps/simple-trading-app/src/app/hooks/use-market-positions.ts:61]: suggestion (non blocking): Use BigNumber?

**Review by mattrussell36 — APPROVED** [review]

**Review by sam-keen — APPROVED** [review]

File: apps/simple-trading-app/src/app/hooks/use-market-positions.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import { gql, useQuery } from '@apollo/client';
+import type {
+  marketPositions,
+  marketPositionsVariables,
+} from './__generated__/marketPositions';
+
+const MARKET_POSITIONS_QUERY = gql`
+  query marketPositions($partyId: ID!) {
```
[the remark this answers is not in the document]
Comment (macqbat) [reply apps/simple-trading-app/src/app/hooks/use-market-positions.ts]: done

File: apps/simple-trading-app/src/app/hooks/use-order-margin.ts [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+import {
+  VegaWalletOrderSide,
+  VegaWalletOrderTimeInForce,
+  VegaWalletOrderType,
+} from '@vegaprotocol/wallet';
+import { addDecimal, formatNumber } from '@vegaprotocol/react-helpers';
+import useMarketPositions from './use-market-positions';
+
+export const ESTIMATE_ORDER_QUERY = gql`
+  query estimateOrder(
```
[the remark this answers is not in the document]
Comment (macqbat) [reply apps/simple-trading-app/src/app/hooks/use-order-margin.ts]: done

File: apps/simple-trading-app/src/app/hooks/use-market-positions.ts (hunk 2 above)
Comment (macqbat) [reply apps/simple-trading-app/src/app/hooks/use-market-positions.ts]: done - changed to BigNumber

File: apps/simple-trading-app/src/app/hooks/use-order-closeout.ts (hunk 1 above)
Comment (macqbat) [reply apps/simple-trading-app/src/app/hooks/use-order-closeout.ts]: agg.openVolume

**Pull request merged**